code/modules/surgery/organ_manipulation.dm

```diff
@@ -36,7 +36,7 @@
 
 /datum/surgery/organ_manipulation/Destroy()
 	if(QDELETED(target) || !HAS_TRAIT(target, TRAIT_FISHING_SPOT))
-		return
+		return ..()
```

Comment:
why does the lint not catch this wtf

Reply:
should_call_parent only checks if there is a parent call anywhere in the proc body, it doesnt check every possible path.

Reply:
disgusting